browser/js/app.js

```diff
@@ -65,4 +73,7 @@ app.run(function($rootScope, AuthService, $state) {
 
     });
 
+    //add lodash to angular view
+    $rootScope._ = window._;
```

Comment:
Unless you're using `_` all over your views, I'd recommend making `_` a value provider instead of a property on the `$rootScope`. Something like `app.value('_', window._)` and then you can inject it into other factories, controllers, etc as needed.